deploy-vm.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+
+if ! [[ $(sudo which docker) && $(sudo docker --version) ]]; then   #[ -x "$(command -v docker)" ];
+    # Install requirements (docker)
+    echo "### Removing existing docker libs"
+    sudo apt-get remove docker docker-engine docker.io containerd runc -y
+    sudo apt-get update -y
+
+    echo "### Installing docker requirements"
+    sudo apt-get install \
+        apt-transport-https \
+        ca-certificates \
+        curl \
+        gnupg-agent \
+        software-properties-common -y
+
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+    sudo add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+
+    sudo apt-get update -y
+
+    echo "### Installing docker"
+    sudo apt-get install docker-ce docker-ce-cli containerd.io -y
+
+    echo "### Finished installing docker. Check docker version"
+    sudo docker --version
+
+fi
+
+if ! [[ $(sudo which docker-compose) && $(sudo docker-compose --version) ]]; then   #[ -x "$(command -v docker)" ];
+
+    echo "### Installing docker-compose"
+    sudo curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    sudo chmod +x /usr/local/bin/docker-compose
+
+    echo "### Finished installing docker-compose. Check docker-compose version"
+    sudo docker-compose --version 
+
+fi
+
+touch .env
+
+read -p "Enter Nodejs (frontend) domain (evalai.example.com) : " FRONTEND_DOMAIN
+echo "FRONTEND_DOMAIN=$FRONTEND_DOMAIN" > .env
+
+read -p "Enter Django (backend) domain (evalapi.example.com) : " BACKEND_DOMAIN
+echo "BACKEND_DOMAIN=$BACKEND_DOMAIN" >> .env
+
+DOCKER_COMPOSE_FILE=""
+
+read -p "Do you want to enable automatic HTTPs with certbot ? (y/N) " decision
+if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
+
+    # Normal HTTP
+    DOCKER_COMPOSE_FILE="docker-compose-vm-http.yml"
```

Comment:
I assume that we are only testing this with EC2 ubuntu AMI only. So lets change the name of the file to `docker-compose-vm-http.yml`

Reply:
Also, I don't think we should put these two docker-compose files in the root folder since it is already overwhelmed. Let's move these to `scripts/ec2_deployment/`

Reply:
Lets also change the name of the docker compose files to `docker-compose-http.yml` and `docker-compose-https.yml`